spec/argv_spec.rb

```diff
@@ -53,6 +53,13 @@ module CLAide
         @argv.remainder.should == %w(--flag ARG1 ARG2 --no-other-flag)
       end
 
+      it 'allows mutliple values for the same option' do
+        @argv = ARGV.new %w(--ignore=foo --ignore=bar)
+        @argv.option('ignore').should == 'bar'
+        @argv.option('ignore').should == 'foo'
+        @argv.option('ignore').should.be.nil
+      end
```

Comment:
How is this gonna be used? Would it not be more useful if there were a variant that would return an array of all values for 1 option key?